types/index.d.ts

```diff
@@ -0,0 +1,17 @@
+import _Vue, { PluginFunction } from "vue";
+
+// augment typings of Vue.js
+import "./vue-injections";
+
+export interface SmoothScrollOptions {
+  scrollTo: Element;
+  duration?: number;
+  offset?: number;
+  container?: Element;
```

Comment:
`container` can be a string as well: https://github.com/Yuliang-Lee/vue2-smooth-scroll/blob/master/src/index.js#L44